submit_and_compare/tests.py

```diff
@@ -44,19 +43,23 @@ def test_student_view(self):
         student_view_html = self.student_view_html()
         self.assertIn(self.xblock.display_name, student_view_html)
         self.assertIn(
-            self.xblock._get_body(self.xblock.question_string),
+            SubmitAndCompareXBlock._get_body(self.xblock.question_string),
             student_view_html,
         )
         self.assertIn(self.xblock._get_problem_progress(), student_view_html)
 
     def test_studio_view(self):
+        # pylint: disable=protected-access
```

Comment:
I'm guessing this was needed to access the value with the leading underscore?
Assuming so, we can remove this once we make the call public (see other comments).